HACKING.md

```diff
@@ -8,27 +15,13 @@ tutorials.ubuntu.com is built with Google's Polymer, using web components. You c
 - https://www.webcomponents.org/community/articles/why-web-components
 
 
-## Adding tutorials
-Tutorials are built via Google Docs and we will be adding markdown support.
+### Our build tools
 
-### `codelabs` command
+Our tooling used to serve and generate this project is available at our [Ubuntu tutorial deployment repository](https://github.com/ubuntu/tutorial-deployment).
 
-We are using a binary built from the ubuntu/codelabs repository. This wraps Google's `claat` command (Look below for more information). The source code is at: https://github.com/ubuntu/codelabs/tree/master/tools/codelabs
+We also make use of a `./run` command that we generate with our [generator-canonical-webteam Yeoman generator](https://github.com/canonical-webteam/generator-canonical-webteam)
```

Comment:
Hum, I find this confusing. It will make people think they need to download the binaries from those project. You should do something like for tutorial-deployment, with the wrapper scripts in bin/ pulling latest release IMHO.